frontend/app/add-user/page.tsx

```diff
@@ -20,17 +20,21 @@ const Register: React.FC = () => {
         }
 
         try {
-            const response = await axios.post(`${window.location.origin}/habitat/api/node/users`, {
-                email,
-                handle,
-                password
-            }, {
+            // TODO: this should just pass through to the PDS xrpc endpoint -- unfortunately there's a bit more node setup that happens.
```

Comment:
yeah we should also use https://www.npmjs.com/package/@atproto/xrpc

Reply:
leaving this as a follow up